angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.ts

```diff
@@ -23,7 +23,13 @@ import {
         >
           <div
             *ngIf="globalLinks"
-            class="sprk-o-Stack__item sprk-o-Stack__item--three-tenths@m sprk-o-Stack sprk-o-Stack--misc-b sprk-o-Box sprk-u-prh"
+            class="
+              sprk-o-Stack__item
+              sprk-o-Stack__item--three-tenths@m
+              sprk-o-Stack
+              sprk-o-Stack--misc-b
+              sprk-c-Footer__global-links
```

Comment:
For footer, i removed the padding util and added container sections for the global links, local links and award.